bdb/rep.c

```diff
@@ -3079,6 +3083,13 @@ static int bdb_wait_for_seqnum_from_all_int(bdb_state_type *bdb_state,
             rc = bdb_wait_for_seqnum_from_node_int(bdb_state, seqnum,
                                                    nodelist[i], 1000, __LINE__);
 
+            if (bdb_lock_desired(bdb_state)) {
```

Comment:
Short-circuit getting the base time in wait-for-seqnum if the bdb lock is desired.  This overlaps with a PR from Dorin who solved the same issue in a different way.